app/views/world_wide_taxons/accordion.html.erb

```diff
@@ -25,6 +28,15 @@
               data_attributes: {
                 "track-count": "accordionSection",
                 id: "section-panel-#{id}-#{index + 1}",
+                ga4: {
+                  event_name: 'select_content',
+                  type: 'accordion',
+                  text: taxon.title,
+                  index: index + 1,
+                  index_total: number_of_accordion_sections,
+                  section: 'nil',
+                  action: 'nil',
```

Comment:
Where `nil` values are required they can just not be passed. The schema holds all of the possible attribute keys with `null` values by default, and anything read from here is written over those values. This means that anything that should be `null` can just be omitted.